core/class-shoutem-api-caching.php

```diff
@@ -87,13 +87,13 @@ public function use_cache( $method, $params ) {
 		}
 
 		if ( ! $use_cache ) {
-			return $method[0]->$method[1]($params);
+			return call_user_func( [ $method[0], $method[1] ], $params );
```

Comment:
Fix for `Indirect access to variables, properties and methods will be evaluated strictly in left-to-right order since PHP 7.0. Use curly braces to remove ambiguity. (PHPCompatibility.Variables.NewUniformVariableSyntax.Found)`